include/ignition/physics/SphereShape.hh

```diff
@@ -92,6 +95,12 @@ namespace ignition
 
         public: using ShapePtrType = SphereShapePtr<PolicyT, FeaturesT>;
 
+        /// \brief Rigidly attach a SphereShape to this link.
+        /// \param[in] _radius
```

Comment:
Just noticed `_name` is missing, but I think this is not introduced by the backport